libcontainer/system/kernelparam/lookup.go

```diff
@@ -0,0 +1,52 @@
+package kernelparam
+
+import (
+	"io/fs"
+	"strings"
+	"unicode"
```

Comment:
I _think_ that kernel boot parameters is guaranteed to be in ASCII and not UTF-8, so maybe no `unicode` usage is required. Have to check that though.

Reply:
fixed